src/tests/adding.md

```diff
@@ -339,8 +339,8 @@ The error levels that you can have are:
 
 ## Revisions
 
-Certain classes of tests support "revisions" (as of February 2018 <!-- date:
-2018-02 -->, this includes compile-fail, run-fail, and incremental, though
+Certain classes of tests support "revisions" (as of February 2021 <!-- date:
+2021-02 -->, this includes compile-fail, run-fail, and incremental, though
 incremental tests are somewhat different). Revisions allow a single test file to
```

Comment:
Need someone knowledgeable with the testsuite to check on this — I've never used revisions before.

Reply:
IIUC revisions are multiple versions of a same test, but it can change more things than just the expected behavior of the program. It can for example change the expected UI, and sets `cfg` flags.

Here's an example of a test that heavily uses revisions: https://github.com/rust-lang/rust/blob/0d97f7a96877a96015d70ece41ad08bb7af12377/src/test/ui/hr-subtype/hr-subtype.rs

Reply:
I think so - it's slightly hard to tell because compiletest enforces which tests *can't* have revisions, not vice-versa, but it looks like these are the only three where the check is omitted (I looked with `rg revisions src/tools/compiletest`).

Reply:
I meant check that the list of test types that support revisions is up-to-date.

Reply:
Okay, I'll assume it's up-to-date then. It's fine if this section is not fully up to date anyway, since it's not that important.